content/new-in-22.md

```diff
@@ -255,9 +256,25 @@ Provide ways to help users navigate, find content, and determine where they are.
 
 Make it easier for users to operate functionality through various inputs beyond keyboard.
 
-### 2.5.7 Dragging Movements (AA)
-<p class="persona"><a href="https://www.w3.org/WAI/people-use-web/user-stories/#retiree">Retiree</a> with hand tremor:</p>
-<div class="quotes">
+### [AG version] 2.5.7 Dragging Movements (AA)
+
+<section class="brief">
+<p>In brief:</p>
+<dl>
+         <dt>Objective</dt><dd>Don't rely on dragging for user actions</dd>
+         <dt>Author task</dt><dd>Provide a simple pointer alternative to any action that involves dragging</dd>
+         <dt>Key beneficiaries</dt><dd>Users with some physical disabilities</dd>
+      </dl>
+ </section>
+
+### [EOWG draft] 2.5.7 Dragging Movements (AA)
+<p>In brief:</p>
+   <ul>
+      <li><p><span class="whatwhy">What to do: </span>For any action that involves dragging, provide a simple pointer alternative.</p></li>
```

Comment:
```suggestion
      <li><p><span class="whatwhy">What to do: </span>Provide a simple pointer alternative to any action that involves dragging.</p></li>
```

Reply:
Mike and Shawn discussed. Agreed to leave as is.